### PR TITLE
Aro migration devhub content

### PR DESCRIPTION
**Pull request by patricksimonian**
## Summary
Added aro migration docs :)

### CODE REVIEW

**Comment by patricksimonian** [comment]
@deploy-tron help

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to dev

File: app-web/topicRegistry/cloud-migration.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        "sourceProperties": {
+          "url": "https://github.com/bcgov/devhub-resources",
+          "owner": "bcgov",
+          "repo": "devhub-resources",
+          "branch": "aro-migration",
+          "files": [
+            "resources/aro/working-in-multiple-clusters.md",
+            "resources/aro/exploring-available-images.md",
+            "resources/aro/finding-app-migration-painpoints.md",
+            "resources/aro/migrating-rhel-images.md
```
Comment (ShellyXueHan) [inline app-web/topicRegistry/cloud-migration.md:17]: missing quote

**Review by ShellyXueHan — CHANGES_REQUESTED** [review]

File: app-web/topicRegistry/cloud-migration.md (hunk 1 above)
Comment (patricksimonian) [reply app-web/topicRegistry/cloud-migration.md:17]: oh shoot! good eye 🤠

Comment (patricksimonian) [inline app-web/topicRegistry/cloud-migration.md:17]:
```suggestion
            "resources/aro/migrating-rhel-images.md"
```

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to dev

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to prod

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to prod

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to test

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to dev

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to test

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to prod

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to dev

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to test

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to prod

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to test

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to prod

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to staging

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to prod

**Review by ShellyXueHan — APPROVED** [review]

**Pull request merged**